### ISSUE TITLE
No Sound output no mic input

### ISSUE THREAD

**Issue by Kenitilas** [playfultechnology/audio-guestbook#79]
Hey all together,

I built the audiobook for my upcoming wedding, but it’s not working.
First up I updated the software yesterday, and it’s showing me 256 blocks at the start in debug. I checked all my wirings and couldn’t find any errors and resoldered every connection and checked the power with the multimeter (5V to ground is present). The software seems to work as the controller waits for the greeting.wav to be played and then starts a recording. But the recording is empty but with the correct length. I also can’t hear the greeting on my phone. 
What I already done:
•	I plugged the speaker to my smartphone jack now I can hear to music without problems. 
•	I tested another microphone I had laying at home but still no sound on my recordings.
•	I tried to record with power from my laptop and with power from a powerbank same results as before
•	I changed the cables from the mic multiple times with no changes
I am really done with my knowledge and need your help! For me it seems the audioshield is doing nothing and may be broken but I don’t have any replacement parts here. This would be the last thing to change if no one has an idea.
Thanks!

**Comment by S41m** [comment]
Hey Kenitilas,

i have the same issue with my project... but i´ve already replaced the audio shield without luck.

Did you already find a solution to this?

Greetings

**Comment by Kenitilas** [comment]
I actually could fix it last night but hadn’t the time left to verify everything… 
I’ve read in a forum or in this repo about a possible ground loop (couldn’t find the post to credit him). So I dismantled my phone again and checked all my cable soldering. I had routed the GND of the phone twice to the board and also reversed the poles of the playback button (GND to PIN). After removing the second GND and soldering the button properly, my phone worked! At least at first quick glance
